src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -1012,6 +1012,22 @@ public static void onEmptyLeftClick(EntityPlayer player, ItemStack stack)
         MinecraftForge.EVENT_BUS.post(new PlayerInteractEvent.LeftClickEmpty(player, stack));
     }
 
+    public static net.minecraftforge.event.entity.player.CriticalHitEvent getCriticalHit(EntityPlayer player, Entity target, boolean vanillaCritical, float vanillaDefault)
+    {
+        if (!vanillaCritical)
+            vanillaDefault = 1F; //So mods can easily check if vanilla say critical or not. 
+        net.minecraftforge.event.entity.player.CriticalHitEvent hitResult = new net.minecraftforge.event.entity.player.CriticalHitEvent(player, target, vanillaDefault);
+        net.minecraftforge.common.MinecraftForge.EVENT_BUS.post(hitResult);
```

Comment:
This is ForgeHooks, not part of the patches, so these should be imported at the top instead of writing them out here.